flask_images/core.py

```diff
@@ -474,6 +474,9 @@ def resized_img_tag(path, **kw):
 
 def resized_img_src(path, **kw):
     self = current_app.extensions['images']
-    return self.build_url(path, **kw)
+    try:
+        return self.build_url(path, **kw)
+    except:
+        return 'https://storage.googleapis.com/assets.getwellio.com/mandoline.cephalo.ai/mandoline-no-image.png'
```

Comment:
Instead of using a hard coded path, let's take in a kw argument with a path.

This library is pretty general (and publicly available).